core/shared/platform/windows/win_clock.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (C) 2023 Amazon Inc.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include "platform_api_vmcore.h"
+#include <winternl.h>
+
+#define NANOSECONDS_PER_SECOND 1000000000ULL
+#define NANOSECONDS_PER_TICK 100
+
+static int
+calculate_monotonic_clock_frequency(uint64 *outfrequency)
```

Comment:
```suggestion
calculate_monotonic_clock_frequency(uint64 *out_frequency)
```
please use sneak case.